File: Sourcode/back/Websitedangtintimkiemnhatro/Websitedangtintimkiemnhatro/Models/Account.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using System.Linq;
using System.Threading.Tasks;

namespace Websitedangtintimkiemnhatro.Models
{
    public class Account
    {
        [Key]
        public int Id { get; set; }
        public string Username { get; set; }
        public string Password { get; set; }
        public string Phone { get; set; }
        public bool IsActive { get; set; }
        public bool IsHD { get; set; }
        public Role Role { get; set; }
        [ForeignKey("RoleId")]
        [Required]
        public int RoleId { get; set; }
        public Employee Employee { get; set; }
        public User User { get; set; }
    }
}

File: Sourcode/font/src/app/model/Account.ts
import { Role } from '../model/Role';
import { User } from '../model/User';
import { Employee } from '../model/Employee';

export class Account {
    id: string;
    username: string;
    password: string;
    phone: string;
    isActive: boolean;
    roleId: string;
    role:Role;
    user:User;
    employee:Employee;
    isHD:boolean;
}
